frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.tsx

```diff
@@ -0,0 +1,80 @@
+import cx from "classnames";
+
+import { UpsellHostingUpdates } from "metabase/admin/upsells";
+import { useSetting } from "metabase/common/hooks";
+import CS from "metabase/css/core/index.css";
+import { Flex } from "metabase/ui";
+import type { SettingDefinition, UpdateChannel } from "metabase-types/api";
+
+import { SettingsSetting } from "../SettingsSetting";
+
+import { VersionUpdateNotice } from "./VersionUpdateNotice/VersionUpdateNotice";
+
+const updateChannelSetting = {
+  key: "update-channel",
+  display_name: "Update Channel",
+  type: "select",
+  description:
+    "Select an update channel to customize which Metabase releases you receive notifications for.",
+  defaultValue: "latest",
+  options: [
+    { name: "Stable", value: "latest" },
```

Comment:
Just so I understand how this is expected to work:

Technically, stable could refer to older (stable) releases, right? Will people get notified of the latest stable release on their current version (e.g., when we backport changes to a previous release), or only the latest and greatest release?

Given the value is "latest" here: I'm assuming we only notify on the latest releases (as currently defined), so we might want to clarify that in the UI, just to make sure people don't think that when they see latest in another context that we're referring to an upstream build like nightly or beta.

```suggestion
    { name: "Stable (latest)", value: "latest" },
```
If people can be on a downstream stable release, and will get notified of point releases on that version, the original copy is preferred, as stable won't always mean latest.

Reply:
> stable won't always mean latest

it should. we should never have a beta or nightly build tagged as latest.

Reply:
>  Will people get notified of the latest stable release on their current version (e.g., when we backport changes to a previous release), or only the latest and greatest release?

There is only ever one latest/stable release. If you are on 49.1 today, you will get notified that you should upgrade to 50.28 (even though there exists a 49.25) Similarly, if you are on 50.27, you will be notified to upgrade to 50.28.